### DISCUSSION TITLE
Could not poll the database: SQLSTATE[HY000] [1130]

### DISCUSSION THREAD

**Discussion by btlspace** [firefly-iii/firefly-iii#8409] [General]
### Could not poll the database: SQLSTATE[HY000] [1130]

Hi,
I try to install Firefly III in my Synology. I use docker-compose file to this ripository with .env and .db.env files.
I've a issue when I access to  Firefly to the first launch. 

> Could not poll the database: SQLSTATE[HY000] [1130] Host '172.26.0.4' is not allowed to connect to this MariaDB server (Connection: mysql, SQL: select `id`, `name`, `data` from `configuration` where `name` = is_demo_site and `configuration`.`deleted_at` is null limit 1)

When I check firefly_III_db logs, i can see that

> 2024-01-17 13:28:18 62 [Warning] Aborted connection 62 to db: 'unconnected' user: 'unauthenticated' host: '172.25.0.4' (This connection closed normally without authentication)

**.db.env**

> MYSQL_RANDOM_ROOT_PASSWORD=yes
MYSQL_USER=firefly
MYSQL_PASSWORD=BtA&a@hP!9?BDnDg
MYSQL_DATABASE=firefly

**.env**

> DB_CONNECTION=mysql
DB_HOST=firefly_iii_db
DB_PORT=3306
DB_DATABASE=firefly
DB_USERNAME=firefly
DB_PASSWORD=BtA&a@hP!9?BDnDg
DB_SOCKET=
MYSQL_USE_SSL=false
MYSQL_SSL_VERIFY_SERVER_CERT=true
MYSQL_SSL_CAPATH=/etc/ssl/certs/
MYSQL_SSL_CA=
MYSQL_SSL_CERT=
MYSQL_SSL_KEY=
MYSQL_SSL_CIPHER=

Thank you for your help

**Comment by btlspace** [comment]
Finally i use the Synology MariaDB package like db. It's working.

**Reply by JC5** [reply]
Glad to hear it works!

> Host '172.26.0.4' is not allowed to connect to this MariaDB server

This feels like some sort of access rights issue, but I'm not sure where it came from.

**Comment by cybertiger19** [comment]
I'm currently facing the same issue. I did install MariaDB via package manager and no resolve. I've had a lot of issues with this install mainly due to trying to get the importer to work.

**Reply by JC5** [reply]
What's your question? If you have the same error, it seems clear to me:

> Host '172.26.0.4' is not allowed to connect to this MariaDB server 

So you need to make sure that the MariaDB you installed via the package manager actually allows other hosts to connect.
